<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QWDialog</class>
 <widget class="QDialog" name="QWDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>501</width>
    <height>449</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QWDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="flat">
      <bool>false</bool>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QCheckBox" name="chkBoxUnderline">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Underline</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBoxItalic">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Italic</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkBoxBold">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Bold</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QRadioButton" name="rbtnBlack">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Black</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnRed">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbtnBlue">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Blue</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Input(&amp;N):</string>
     </property>
     <property name="buddy">
      <cstring>textEdit</cstring>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit">
     <property name="font">
      <font>
       <family>Microsoft JhengHei UI Light</family>
       <pointsize>10</pointsize>
      </font>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnConfirm">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Confirm</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnCancel">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExit">
        <property name="font">
         <font>
          <family>Microsoft JhengHei UI Light</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Exit</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>407</y>
    </hint>
    <hint type="destinationlabel">
     <x>199</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnConfirm</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>114</x>
     <y>414</y>
    </hint>
    <hint type="destinationlabel">
     <x>113</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>QWDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>273</x>
     <y>410</y>
    </hint>
    <hint type="destinationlabel">
     <x>289</x>
     <y>444</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
